CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required (VERSION 2.8.12)
 project (PIO C Fortran)
-cmake_policy(VERSION 3.5.2)
+#cmake_policy(VERSION 3.5.2)
```

Comment:
@jayeshkrishna that is a good question. Actually I had just set it to the version of doxygen that I have installed on my local machine.